pom.xml

```diff
@@ -39,6 +38,9 @@
     </properties>
 
     <build>
+
+    <finalName>${project.artifactId}</finalName>
```

Comment:
Please indent it properly and remove useless whitelines above and below

Reply:
removed whitelines :)